tests/gtest/aviftest_helpers.h

```diff
@@ -23,6 +26,20 @@ class AvifRwData : public avifRWData {
   ~AvifRwData() { avifRWDataFree(this); }
 };
 
+class AvifRgbImage : public avifRGBImage {
+ public:
+  AvifRgbImage(const avifImage* yuv, int rgbDepth, avifRGBFormat rgbFormat);
+  ~AvifRgbImage() { avifRGBImageFreePixels(this); }
+};
+
+//------------------------------------------------------------------------------
+// Samples and images
+
+struct RgbChannelOffsets {
```

Comment:
Nit: Add a comment for this struct. We can just adapt the following comment from a previous version of this pull request:

```
// Maps from avifChannel to sample position in avifRGBFormat.
uint32_t avifChannelOffset(avifRGBFormat format, avifChannel channel);
```